scripts/globals/instance.lua

```diff
@@ -1,10 +1,308 @@
+        -- { 0, { 0, 23,  0 } }, -- Distorter of Time
+        -- Campaign Ops
+        -- Brave Dawn I (Bastok)
+        -- Brave Dawn II (Bastok)
+        -- Brave Dawn III (Bastok)
+        -- By Light of Fire I (Bastok)
+        -- Granite Rose I (Bastok)
+        -- Granite Rose II (Bastok)
+        -- Granite Rose III (Bastok)
+        -- Pit Spider I (Bastok)
+        -- Pit Spider II (Bastok)
+    },
+
+    [xi.zone.GHOYUS_REVERIE] =
+    {
+        -- Doomvoid - Lambton Worm
+        -- Doomvoid - Serket
+        -- A Feast for Gnats
+        -- A Manifest Problem
+        -- In a Haze of Glory
+        -- Sins of the Mothers
+        -- Campaign Ops
+        -- Brave Dawn I (Windurst)
+        -- Brave Dawn II (Windurst)
+        -- Brave Dawn III (Windurst)
+        -- Granite Rose I (Windurst)
+        -- Granite Rose II (Windurst)
+        -- Pit Spider I (Windurst)
+        -- Pit Spider II (Windurst)
+    },
+
+    [xi.zone.MAQUETTE_ABDHALJS_LEGION_A] =
+    {
+        -- Hall of An
+        -- Hall of Ki
+        -- Hall of Im
+        -- Hall of Muru
+        -- Hall of Mul
+    },
+
+    [xi.zone.RALA_WATERWAYS_U] =
+    {
+        -- {  0, 0 }, -- Endeavoring to Awaken
+        -- {  1, 0 }, -- Endeavoring to Awaken
+        -- -- Blank
+        -- {  3, 0 }, -- Behind the Sluices
+        -- {  4, 0 }, -- Stonewalled
+        -- {  5, 0 }, -- The Gates
+        -- {  6, 0 }, -- Saved by the Bell
+        -- {  7, 0 }, -- Quiescence
+        -- {  8, 0 }, -- The Charlatan
+        -- {  9, 0 }, -- Yggdrasil Beckons
+        -- { 10, 0 }, -- Yggdrasil Beckons
+        -- { 11, 0 }, -- Watery Grave
+        -- { 12, 0 }, -- Mistress of Ceremonies
+        -- { 13, 0 }, -- A Barrel of Laughs
+        -- { 14, 0 }, -- Sinister Reign
+        -- { 15, 0 }, -- The Ygnas Directive 6
+        -- { 16, 0 }, -- Skirmishes
+        -- { 17, 0 }, -- Fractures
+        -- { 18, 0 }, -- Alluvion skirmishes
+        -- { 19, 0 }, -- The Silent Forest
+        -- { 20, 0 }, -- Wind of Eternity
+    },
+
+    [xi.zone.YORCIA_WEALD_U] =
+    {
+
+    },
+
+    [xi.zone.CIRDAS_CAVERNS_U] =
+    {
+
+    },
+
+    [xi.zone.OUTER_RAKAZNAR_U] =
+    {
+
+    },
+
+    [xi.zone.DYNAMIS_SAN_DORIA_D] =
+    {
+
+    },
+
+    [xi.zone.DYNAMIS_BASTOK_D] =
+    {
+
+    },
+
+    [xi.zone.DYNAMIS_WINDURST_D] =
+    {
+
+    },
+
+    [xi.zone.DYNAMIS_JEUNO_D] =
+    {
+
+    },
+}
+
+xi.instance.onTrade = function(player, npc, trade)
+end
+
+xi.instance.onTrigger = function(player, npc, instanceZoneID, csid)
+    local zoneLookup = xi.instance.lookup[instanceZoneID]
+
+    -- TODO: Extract player eligibility
+
+    player:startEvent(csid, unpack(zoneLookup[1][2]))
+end
+
+xi.instance.onEventUpdate = function(player, csid, option)
+end
+
+xi.instance.onEventFinish = function(player, csid, option)
+end
+
+local function setInstanceLastTimeUpdateMessage(instance, players, remainingTimeLimit, texttable)
```

Comment:
texttable vs textTable?

Reply:
Yeah, that was just copy/pasting what was already there - I'll probably turn it into `text` when I come back around to clean